tripy/docs/conf.py

```diff
@@ -178,40 +187,40 @@ def process_docstring(app, what, name, obj, options, lines):
                     ":returns:" in doc
                 ), f"For: {obj}, return value is not documented. Please ensure you've included a `Returns:` section"
 
-    # new docstring logic:
-    # first figure out if we should it is the new docstring
+    # New docstring logic:
+    # First figure out if we should it is the new docstring.
     if name.split(".")[-1] in TYPE_VERIFICATION.keys():
         cleaned_name = name.split(".")[-1]
         add_text_index = -1
         for index, block in enumerate(blocks):
             if re.search(r".. code-block::", block):
-                type_dict = TYPE_VERIFICATION[cleaned_name][1]["types"]
+                type_dict = TYPE_VERIFICATION[cleaned_name].dtypes
                 blocks.insert(index, "Type Constraints:")
                 index += 1
+                # Add the dtype constraint name and the dtypes that correlate. Remove excluded dtypes.
                 for type_name, dt in type_dict.items():
-                    blocks.insert(index, f"    - {type_name}: " + ", ".join(dt))
+                    blocks.insert(
+                        index,
+                        f"    - **{type_name}**: :class:`" + "`, :class:`".join(set(dt) - {"int4", "float8"}) + "`",
```

Comment:
Why do we manually remove int4/float8 here?

Reply:
Int4 and float8 still have a lot of bugs so I have been excluding them from verification for now.

Reply:
Maybe we should make the tests XFAIL but not remove them from the docs? Or if we don't expect them to be fixed soon, let's not claim support for them in the first place?

Reply:
The issue with int4 and float8 is that they are unreliable since they pass for some uses cases but not all of them. For example, casting from float8 works for every dtype except for float8->int8. I can maybe add a case that notes these cases in the documentation (add a portion of documentation that says current dtype bugs/exceptions). Currently the documentation removes int4 and float8 meaning that we are not claiming support for int4 or float8 operations.

Reply:
Agreed with @pranavm-nvidia , lets make it XFAIL, and maybe also catch them in our frontend and report to the user that this is not supported.